### PR TITLE
support epoch_now_in_ms

### PR DESCRIPTION
**Pull request by chunyan**

### CODE REVIEW

File: charlatan/file_format.py [hunk 1]
```diff
@@ -73,6 +73,19 @@ def get_now():
...
+    def epoch_now_in_ms_constructor(loader, node):
+        """Return a function that returns the current epoch in milliseconds.
+
+        :rtype: int
+        """
+        delta = get_timedelta(loader.construct_scalar(node))
+
+        def get_now():
+            return int(1000 * datetime_to_epoch_timestamp(
+                datetime.datetime.utcnow() + delta))
```
Comment (erikformella) [inline charlatan/file_format.py]: shouldn't you add the delta after the conversion to ms? i think the yaml files will be harder to read if the delta is in seconds for a field that is in ms.

**Comment by erikformella** [comment]
please add a test for this functionality. there should be existing ones for the other constructors.

File: charlatan/file_format.py [hunk 2]
```diff
@@ -5,7 +5,11 @@
 import yaml
 from yaml.constructor import Constructor
 
-from charlatan.utils import get_timedelta, datetime_to_epoch_timestamp
+from charlatan.utils import (
```
Comment (charlax) [inline charlatan/file_format.py]: I prefer duplicating lines like this, to allow for quick sorting:

```
from charlatan.utils import get_timedelta
from charlatan.utils import ...
```

File: charlatan/tests/test_file_format.py [hunk 3]
```diff
@@ -11,7 +11,10 @@
 
 from charlatan import testing
 from charlatan import file_format
-from charlatan.utils import datetime_to_epoch_timestamp
+from charlatan.utils import (
```
Comment (charlax) [inline charlatan/tests/test_file_format.py]: Same comment about keeping import style consistent.

**Comment by charlax** [comment]
Left two minor comments, can you fix them and I'll merge immediately - thanks :)

**Pull request merged**